katcp/server.py

```diff
@@ -2325,7 +2326,7 @@ def set_log_level(self, level):
         """
         self._log_level = level
         if self._python_logger:
-            level = PYTHON_LEVEL.get(level)
+            level = self.PYTHON_LEVEL.get(level)
```

Comment:
can we make this more safe:

Where you say: 
 level = self.PYTHON_LEVEL.get(level)
please change that to:
 level = self.PYTHON_LEVEL.get(level, logging.NOTSET)

So if an strange log level is requested KATCP will not blow up but will just log everything, since it has no filter. An example is the log level ALL as defined in the list but not implemented in the dict.